features/environment.py

```diff
@@ -412,6 +412,59 @@ def before_scenario(context: Context, scenario: Scenario):
             )
 
 
+def after_scenario(context, scenario):
+    """Collect the coverage files after the scenario is run."""
+    cov_dir = os.path.join(context.pro_config.artifact_dir, "coverage")
+    if not os.path.exists(cov_dir):
+        os.makedirs(cov_dir)
+
+    inner_dir = os.path.join(
+        datetime.datetime.now().strftime("%Y-%m-%d"),
+        "{}".format(os.path.basename(scenario.filename.replace(".", "_"))),
+    )
+    new_artifacts_dir = os.path.join(
+        cov_dir,
+        inner_dir,
+    )
+    if not os.path.exists(new_artifacts_dir):
+        os.makedirs(new_artifacts_dir)
+    if hasattr(context, "machines") and SUT in context.machines:
+        try:
+            scenario_name = (
+                os.path.basename(scenario.filename.replace(".", "_"))
+                + "_"
+                + str(scenario.line)
+            )
+            cov_filename = ".coverage.{}".format(scenario_name)
+            context.machines[SUT].instance.execute(
+                [
+                    "bash",
+                    "-c",
+                    "mv .coverage /tmp/{cov_filename}".format(
+                        cov_filename=cov_filename
+                    ),
```

Comment:
I don't think this needs to execute in bash - or am I missing something?
```suggestion
                    "mv",
                    ".coverage",
                    "/tmp/{}".format(cov_filename),
```

Reply:
Also, I'd prefer to use an absolute path to the .coverage file